internal/resources/slo/resource_slo.go

```diff
@@ -51,6 +51,12 @@ Resource manages Grafana SLOs.
 				Description:  `Description is a free-text field that can provide more context to an SLO.`,
 				ValidateFunc: validation.StringLenBetween(0, 1024),
 			},
+			"folder": {
```

Comment:
```suggestion
			"folder_uid": {
```

Can we call it that way to make it clear that we expect the `grafana_folder.<>.uid` attribute?

Reply:
hi @julienduchesne thank your for your review. It make sense, I've updated it here: 7bb5292da7c52d4fbef79a4f1359ef7dadb48ad3

Reply:
going to fix the test locally

Reply:
@julienduchesne is it mandatory to add tests for the `folder_uid` input? If that's the case, I'm thinking to follow [resource_dashboard_test](https://github.com/grafana/terraform-provider-grafana/blob/main/internal/resources/grafana/resource_dashboard_test.go#L316-L336) implementation by creating folder first